config/admin-functions/rekey_via_tpm.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Same comment as on the vxsuite-build-system PR: Do we want to add a `set -euo pipefail` to ensure that this script fails if any individual command fails?